app/styles/custom-components/_vlc-panel-layout.scss

```diff
@@ -2,32 +2,53 @@
    vl-panel-layout
    ========================================================================== */
 
-.vlc-panel-layout {
-  display: flex;
-  height: 100%;
-  // Firefox fix for scroll with flex-direction: column
-  min-height: 0;
-}
-
-.vlc-panel-layout-agenda__detail {
-  width: 100%; // Fix a flexbox issue
-}
-
-.vlc-panel-layout__agenda-history {
-  height: 100%;
-}
-
-.vlc-panel-layout__agenda-items {
-  flex: 1 1 20%;
-  height: 100%;
-  min-width: 320px;
-  max-width: 480px;
-  background: $porcelain;
-  border-right: 1px solid $ghost;
-}
-
-.vlc-panel-layout__main-content {
-  flex: 1 1 auto;
-  height: 100%;
-  min-width: 0;
-}
+   .vlc-panel-layout {
```

Comment:
@evajacobs can you fix the indentation here? There are two space at the beginning of this line that shouldn’t be there :-)